corehq/apps/data_dictionary/static/data_dictionary/js/data_dictionary.js

```diff
@@ -125,7 +126,16 @@ hqDefine("data_dictionary/js/data_dictionary", [
         };
 
         self.deprecateProperty = function () {
-            self.deprecated(true);
+            if (!self.isGeospatialProperty()) {
```

Comment:
What if the project using the data dictionary does not make use of the geospatial feature? Will the modal still appear?

Reply:
If the geospatial feature is disabled then the project wouldn't receive a modal, even if it's currently set as the case property in the domain's `GeoConfig`. When the FF is disabled, then `isGeospatialProperty()` defaults to `False`.